CMakeLists.txt

```diff
@@ -450,6 +453,21 @@ else(RDK_TEST_COVERAGE)
  endif()
 endif(RDK_TEST_COVERAGE)
 
+if(RDK_BUILD_UBSAN)
```

Comment:
I'd rather not add these as global options to the project since they don't work across all compilers and are something that can easily be added by developers via either `CXXFLAGS` directly or by editing `CMAKE_CXX_FLAGS`.

Reply:
Makes sense. I've reverted the additon.